### PR TITLE
[docs] How to install Deckhouse in a private environment

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Added an article in the getting started about how to install Deckhouse in a private environment.

## Changelog entries
```changes
section: docs
type: fix
summary:  Getting started - installing Deckhouse in a private environment.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by Zhbert — COMMENTED** [review]
It's all good. Literally made a couple of small edits.

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 1]
```diff
@@ -21,7 +23,11 @@ You will need:
    The computer from which the installation will be performed.{% unless page.platform_type == "existing" %} It is only needed to run the Deckhouse installer, and will not be part of the cluster.{% endunless %}
 
    Requirements:
-   - HTTPS-access to the container registry `registry.deckhouse.io` (it is also possible to use a [third-party registry](/{{ page.lang }}/documentation/v1/deckhouse-faq.html#how-do-i-configure-deckhouse-to-use-a-third-party-registry));
+{% if page.platform_code == "bm-private" %}
+   - Access to a proxy registry ([read more](../../documentation/v1/deckhouse-faq.html#how-do-i-configure-deckhouse-to-use-a-third-party-registry) about setting them up) or to a private container image registry containing Deckhouse images;
+{%- else %}
+   - HTTPS access to the `registry.deckhouse.io` container image registry (it is also possible to use a [third-party registry](../bm-private/step2.html));
+{%- endif %}
```
Comment (z9r5) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:30]:
```suggestion
MAKE SOME CHANGES
{% if page.platform_code == "bm-private" %}
   - Access to a proxy registry ([read more](../../documentation/v1/deckhouse-faq.html#how-do-i-configure-deckhouse-to-use-a-third-party-registry) about setting them up) or to a private container image registry containing Deckhouse images;
{%- else %}
   - HTTPS access to the `registry.deckhouse.io` container image registry (it is also possible to use a [third-party registry](../bm-private/step2.html));
{%- endif %}
```

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# [<ru>] Секция с параметрами Deckhouse.
+deckhouse:
+  # [<en>] The release channel in use.
+  # [<ru>] Используемый канал обновлений.
+  releaseChannel: Stable
+  configOverrides:
+    global:
+      modules:
+        # [<en>] Template that will be used for system apps domains within the cluster
+        # [<en>] e.g., Grafana for %s.example.com will be available as grafana.example.com.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc:47]:
```suggestion
        # [<en>] Template to use for system app domains within the cluster.
        # [<en>] For example, in the case of %s.example.com, Grafana will be mapped to grafana.example.com.
```

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    cniFlannel:
+      # [<en>] Flannel backend, available values are VXLAN (if your servers have L3 connectivity) and HostGW (for L2 networks).
+      # [<ru>] Режим работы flannel, допустимые значения VXLAN (если ваши сервера имеют связность L3) или HostGW (для L2-сетей).
+      # [<en>] You might consider changing this.
+      # [<ru>] Возможно, захотите изменить.
+      podNetworkMode: VXLAN
+  # [<en>] Address of the Docker registry where the Deckhouse images are located.
+  # [<ru>] Адрес Docker registry с образами Deckhouse.
+  imagesRepo: <IMAGES_REPO_URI>
+  # [<en>] A special string with your token to access Docker registry.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc:65]:
```suggestion
  # [<en>] A special string with your token to access the Docker registry.
```

File: docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+# [<en>] Section with the parameters of the bare metal cluster (StaticClusterConfiguration).
+# [<en>] Version of the Deckhouse API.
+# [<ru>] Секция с параметрами bare metal кластера (StaticClusterConfiguration).
+# [<ru>] Используемая версия API Deckhouse.
+apiVersion: deckhouse.io/v1
+# [<en>] Type of the configuration section.
+# [<ru>] Тип секции конфигурации.
+kind: StaticClusterConfiguration
+# [<en>] List of internal cluster networks (e.g., '10.0.4.0/24'), which is
+# [<en>] used for linking Kubernetes components (kube-apiserver, kubelet etc.).
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/bm-private/partials/config.yml.default.ce.inc:84]:
```suggestion
# [<en>] List of internal cluster networks (e.g., '10.0.4.0/24')
# [<en>] for linking Kubernetes components (kube-apiserver, kubelet etc.).
```

File: docs/site/_includes/getting_started/bm-private/partials/user.yml.inc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# [<ru>] Используемая версия API Deckhouse.
+apiVersion: deckhouse.io/v1
+kind: User
+metadata:
+  name: admin
+spec:
+  # [<en>] User e-mail.
+  # [<ru>] E-mail пользователя.
+  email: admin@example.com
+  # [<en>] This is a hash of the password <GENERATED_PASSWORD>, generated  now.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/bm-private/partials/user.yml.inc:30]:
```suggestion
  # [<en>] This is a hash of the newly generated <GENERATED_PASSWORD> password.
```

File: docs/site/_includes/getting_started/bm-private/partials/user.yml.inc [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  # [<en>] Generate your own or use it at your own risk (for testing purposes):
+  # [<en>] echo "<GENERATED_PASSWORD>" | htpasswd -BinC 10 "" | cut -d: -f2
+  # [<ru>] Это хэш пароля <GENERATED_PASSWORD>, сгенерированного сейчас.
+  # [<ru>] Сгенерируйте свой или используйте этот, но только для тестирования:
+  # [<ru>] echo "<GENERATED_PASSWORD>" | htpasswd -BinC 10 "" | cut -d: -f2
+  # [<en>] You might consider changing this.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/bm-private/partials/user.yml.inc:36]:
```suggestion
  # [<en>] You might consider changing it.
```

File: docs/site/_includes/getting_started/global/partials/INSTALL_OTHER.liquid [hunk 7]
```diff
@@ -1,6 +1,9 @@
 {% assign revision=include.revision %}
 
-Use a Docker image to install the **Deckhouse Platform**. It is necessary to transfer configuration files to the container{%- if page.platform_type != 'existing' %}, as well as SSH keys for access to the master nodes{% endif %}. Run on the **[personal computer](step2.html#installation-process)**:
+Use a Docker image to install the **Deckhouse Platform**. It is necessary to transfer configuration files to the container{%- if page.platform_type != 'existing' %}, as well as SSH keys for access to the master nodes{% endif %}.
+{%- if page.platform_code == 'bm-private' %} Log in on the **[personal computer](step2.html#installation-process)** to the container image registry you specified in the previous step, then run:
+{%- else %} Run on the **[personal computer](step2.html#installation-process)**:{% endif %}
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/global/partials/INSTALL_OTHER.liquid:5]:
```suggestion
Use a Docker image to install the **Deckhouse Platform**. It is necessary to transfer configuration files to the container{%- if page.platform_type != 'existing' %} as well as SSH keys for accessing the master nodes{% endif %}.
{%- if page.platform_code == 'bm-private' %} Log in on the **[personal computer](step2.html#installation-process)** to the container image registry you specified in the previous step, then run:
{%- else %} Run on the **[personal computer](step2.html#installation-process)**:{% endif %}
```

File: docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid [hunk 8]
```diff
@@ -11,6 +9,10 @@ This guide covers installing Deckhouse in a **minimal** configuration, with the
 
 After completing all the steps in this guide, you will be able to enable all the modules of interest on your own. Please, refer to the [documentation](/{{ page.lang }}/documentation/v1/) to learn more or reach out to the Deckhouse [community](/en/community/about.html).
 
+{% elsif page.platform_code == "bm-private" %}
+This guide covers deploying a Kubernetes cluster using Deckhouse in a private environment when there is no direct access to a container image registry or deb/rpm package repositories.
+
+Installation in a private environment is generally similar to installation on [bare metal](../bm/step2.html). The only differences are in some additional settings.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA.liquid:15]:
```suggestion
This guide focuses on deploying a Kubernetes cluster using Deckhouse in a private environment when there is no direct access to the container image registry or deb/rpm package repositories.

Installation in a private environment is generally similar to installation on [bare metal](../bm/step2.html). The only difference is in some additional settings.
```

**Review by kirkonru — COMMENTED** [review]
[docs] Some minor edits and rewordings

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**